packages/popover/src/Popover.story.tsx

```diff
@@ -8,13 +8,19 @@ import Popover, { Align, Justify } from '.';
 
 const containerStyle = css`
   position: absolute;
+  min-height: 50px;
+  appearance: none;
+  box-shadow: 0 0 4px #000;
+  background-color: ${colors.gray[5]};
```

Comment:
Nit, but while you're in here...
Can we pick a gray from palette instead of theme? I believe this is the only time that the theme package is used in Popover and if we can swap that over to palette that'd be great as we are trying to deprecate theme. @DesignerDave not sure if you have a preference on which gray?

Reply:
Sure, I just used it because it was already being imported, but it'd be easy enough to switch that over. It should probably just be a devDependency, too, if it's only being used in storybook. Looks like right now it's listed under `dependencies`.

Reply:
👍  Thank you so much